source/dom.js

```diff
@@ -1,4 +1,10 @@
-const div = document.createElement('div');
+import { isBrowserEnv } from './utils';
+
+const fakeWindow = {};
+
+export const global = isBrowserEnv ? window : fakeWindow;
+
+const div = isBrowserEnv ? document.createElement('div') : {};
```

Comment:
Why do we need `fakeWindow`? Maybe it's more convenient to create `div` inside `createNode` method where it's needed? In this case seems `createNode` will now be called on server at all and therefore there's no error in runtime, right?